### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by ciaranschutte**
- some styling fixes
- easier fix for table to reduce complexity in changes

### CODE REVIEW

File: src/app/(post-login)/submission/program/[shortName]/clinical-data/ClinicalEntityDataTable.tsx [hunk 1]
```diff
@@ -655,17 +653,24 @@ const ClinicalEntityDataTable = ({
 			z-index: 1;
 		`;
 
+		const s = css`
```
Comment (demariadaniel) [inline src/app/(post-login)/submission/program/[shortName]/clinical-data/ClinicalEntityDataTable.tsx:656]: Hate to hold you up but I think this deserves a more descriptive name

**Comment by demariadaniel** [comment]
Submitted Data page is not rendering properly, background is missing (confirmed working correctly on `develop`)

Screenshot 2024-01-31 at 11 02 14 AM

**Comment by ciaranschutte** [comment]
@demariadaniel you're right. thank you for taking a look.

**Comment by ciaranschutte** [comment]
@demariadaniel breaking this up into individual PRs

**Pull request closed without merging**